front-packages/akeneo-design-system/src/components/BlockButton/BlockButton.tsx

```diff
@@ -0,0 +1,135 @@
+import React, {isValidElement, ReactNode, Ref, SyntheticEvent} from 'react';
+import styled, {css} from 'styled-components';
+import {AkeneoThemedProps, getColor, getFontSize} from '../../theme';
+import {Override} from '../../shared';
+import {ArrowDownIcon, IconProps} from '../../icons';
+
+type BlockButtonProps = Override<
+  React.ButtonHTMLAttributes<HTMLButtonElement> & React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  {
+    /**
+     * Use when the user cannot proceed or until an input is collected.
+     */
+    disabled?: boolean;
+
+    /**
+     * Function called when the user clicks on the button or hit enter when focused.
+     */
+    onClick?: (event: SyntheticEvent) => void;
+
+    /**
+     * Accessibility label to describe shortly the button.
+     */
+    ariaLabel?: string;
+
+    /**
+     * Define which element is the label of this button for accessibility purposes. Expect a DOM node id.
+     */
+    ariaLabelledBy?: string;
+
+    /**
+     * Define what element is describing this button for accessibility purposes. Expect a DOM node id.
+     */
+    ariaDescribedBy?: string;
+
+    /**
+     * Children of the button.
+     */
+    children?: ReactNode;
+  }
+>;
+
+const getColorStyle = ({disabled}: {disabled: boolean} & AkeneoThemedProps) => {
+  if (disabled) {
+    return css`
+      border-color: ${getColor('grey', 100)};
+      color: ${getColor('grey', 100)};
+    `;
+  }
+  return css`
+    background-color: ${getColor('white')};
+    border-color: ${getColor('blue', 100)};
+    color: ${getColor('blue', 100)};
+  `;
+};
+
+const Container = styled.button<
+  {
+    disabled: boolean;
+  } & AkeneoThemedProps
+>`
+  width: 100%;
+  padding: 14px 20px;
+  border-style: solid;
+  border-width: 1px;
+  border-radius: 2px;
+  display: flex;
+  justify-content: space-between;
+  font-family: inherit;
+  font-size: ${getFontSize('default')};
+  font-weight: 400;
+  outline-style: none;
+  text-decoration: none;
+  cursor: ${({disabled}) => (disabled ? 'not-allowed' : 'pointer')};
+  outline-style: none;
+  text-decoration: none;
+
+  &:focus {
+    box-shadow: 0 0 0 2px ${getColor('blue', 40)};
+  }
+
+  ${getColorStyle}
+`;
+
+const ChildrenContainer = styled.div`
+  display: flex;
+  align-items: center;
```

Comment:
```suggestion
  align-items: center;
  gap: 10px;
```
Just like in https://github.com/akeneo/pim-community-dev/blob/e8642d9427d72b900565bb1716f8144b64e582d7/front-packages/akeneo-design-system/src/components/Button/Button.tsx#L113
<img width="251" alt="image" src="https://user-images.githubusercontent.com/3492179/158545839-d3ed0522-873b-495d-864a-2ca43e80817f.png">